data/map planets.txt

```diff
@@ -382,6 +395,10 @@ planet "Big Sky"
 	description `	The Big Sky government recently financed a major project to genetically engineer a goat species so ravenous that it can control the growth of local vegetation. These "ubergoats" pose so great a threat to the typical biosphere that they are export-controlled as biological weapons.`
 	spaceport `The landing pads here are being gradually consumed by the local plant life. In most places grass and ivy has crept out to cover at least a meter of the pad on all sides, and every small crack or crevice has been colonized by moss and by the roots of larger plants.`
 	spaceport `	In places the forest has been cleared to make roads leading to the nearby farming villages, but in most directions all you see is green. It is a beautiful sight, and oddly peaceful, but you know what a nuisance these plants are to the locals.`
+	spaceport `	A large gathering of locals is congregated around a memorial in the spaceport to commemorate Remembrance Day. The plaque and the surrounding path have been scoured clean of the native flora.`
```

Comment:
This is a little unclear about whether the gathering is commemorating Remembrance Day, or the memorial; it reads like the first (probably) but is probably supposed to be both?

Also, saying a gathering is congregated is repetitive, which makes me feel like I'm missing something the first time I read it.

```suggestion
	spaceport `	A large number of locals have congregated around a memorial in the spaceport in order to commemorate Remembrance Day. The plaque and the surrounding path have been scoured clean of the native flora.`
```